contracts/account/src/contract.rs

```diff
@@ -22,6 +22,45 @@ pub fn instantiate(
     execute::init(deps, env, &mut msg.authenticator.clone())
 }
 
+#[cw_serde]
+pub struct Any {
+    pub type_url: String,
+    pub value: Binary,
+}
```

Comment:
```suggestion
```
cosmwasm_std already contains exactly same structure since 2.0, let's re-use it https://docs.rs/cosmwasm-std/2.1.4/cosmwasm_std/struct.AnyMsg.html